pkg/system/phase4_configuring.go

```diff
@@ -60,6 +60,16 @@ func (r *Reconciler) ReconcilePhaseConfiguring() error {
 		}
 	}
 
+	// update noobaa-core version in reconciler struct
+	r.NBClient.SetAuthToken(r.SecretOp.StringData["auth_token"])
```

Comment:
I think this is redundant

Reply:
didn't work without it for some reason

Reply:
I think I know why. I'll fix it

Reply:
I remove the `SetAuthToken` in phase 3, since it sets an invalid token (secret was not read yet). setting auth token in `ReconcileSecretOp` instead